docs/src/solvers/cma_es.md

```diff
@@ -0,0 +1,29 @@
+# Particle swarm optimization
```

Comment:
This could definitely still be improved.

Reply:
Yes, I haven't worked that much on docs yet.